### PR TITLE
Add Serbian translation for Eclipse Collections website.

### PR DESCRIPTION
**Pull request by TheJavaGuy**
Hello, I have added Serbian translation for Eclipse Collections website. @nikhilnanivadekar I saw there are no PRs but I guessed that somebody must had contributed translations to Spanish, Greek, Norwegian, Russian etc. before. If this is not the right way to contribute I apologise, I couldn’t find any info on this topic.

### CODE REVIEW

**Comment by nikhilnanivadekar** [comment]
@TheJavaGuy previously the website was managed differently, now it is on GitHub. So the PR method is correct. Since I don’t speak Serbian, would you please ask someone to proof read and review the translation. If they comment their approval on the PR, I can merge it.
Thank you very much for your contribution

File: sr-rs-latn/index.html [hunk 1]
```diff
@@ -0,0 +1,675 @@
...
+    <div class="content-section-a">
+
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">Početak rada sa <br>Eclipse Collections</h2>
+                    <p class="lead">Eclipse Collections su daleko najbolje kolekcije za Javu
+                        koje će ti doneti radost u razvoj softvera.
```
Comment (ghost) [inline sr-rs-latn/index.html]: There is a typo here: "u razvoju"

**Review by ghost — APPROVED** [review]

File: sr-rs-latn/index.html (hunk 1 above)
Comment (TheJavaGuy) [reply sr-rs-latn/index.html]: Thank you @RokiShopen, fixed.

**Comment by TheJavaGuy** [comment]
@nikhilnanivadekar We're all set, I fixed the only remark that @RokiShopen had so this is ready to merge.

**Comment by TheJavaGuy** [comment]
@nikhilnanivadekar Are you satisfied with the review?

**Comment by TheJavaGuy** [comment]
@nikhilnanivadekar Please don't let this PR rot.

**Comment by nikhilnanivadekar** [comment]
Sorry missed this. The review is fine. Checking the ECA.

**Comment by nikhilnanivadekar** [comment]
Can you please sign-off the commit? And if you have not signed the ECA please do the same: https://github.com/eclipse/eclipse-collections/blob/master/CONTRIBUTING.md#contributor-license

I’ll include the WCA validation workflow to this repo separately.
Appreciate your patience and sorry for the delay!

**Comment by nikhilnanivadekar** [comment]
ECA is validated. Merging based on the review feedback provided here.

**Comment by TheJavaGuy** [comment]
@nikhilnanivadekar Thank you! I'll set up sign-off for my future commits.

**Pull request merged**